doc/virtualization.md

```diff
@@ -201,3 +201,11 @@ DHCP. User can provide own IP address by editing
 vmq0 interface is disabled in this release. But it can be enabled back
 by un-commenting corresponding line in `/etc/xen/domu.cfg` file in
 Dom0.
+
+## PMU
+Note, this feature is not security supported:
+see https://xenbits.xen.org/xsa/advisory-163.html.
+It's not recommended to use it for production builds
+but it may be  used for development or debugging.
+PMU is disabled by default. It can be enabled by by un-commenting corresponding
+lines in `/etc/xen/domu.cfg` or `/etc/xen/domd.cfg` file in Dom0.
```

Comment:
Could you please add a couple more paragraphs?
1. Describe why do we need pin vCPUs
2. Describe why it is not securely supported

Reply:
Thank you - added descriptions:)